apps/LineRider/install

```diff
@@ -1,19 +1,32 @@
 #!/bin/bash
 
-version=2022.5.3
 #Bot automatically updates this when I make a new release.
+version=2022.5.3
+#Buster version
+version2=2022.5.1
 
+status "Installing Depends... "
 install_packages apt-transport-https dirmngr gnupg ffmpeg mono-devel unzip zip || exit 1
+status_green "Done."
 
 mkdir -p ~/LineRider
 cd ~/LineRider || error "Could not move to folder"
 
+if [[ $(get_codename) == "buster" ]]; then
+  use_version="$version2"
+else
+  use_version="$version"
+fi
+
 #Download LRA-Community-Edition build from repo
-wget -O linerider.zip https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${version}/LineRider.game.auto-release.${version}-LR.zip || error "Could not download linerider"
+status "Downloading LineRider... "
+wget -O linerider.zip https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${use_version}/LineRider.game.auto-release.${use_version}-LR.zip || error "Could not wget LineRider from GitHub!"
 
 #Install
+status "Unzipping... "
 unzip linerider.zip || error "Could not unzip the files"
 rm linerider.zip
+status_green "Done."
```

Comment:
These "Unzipping... " and "Done." messages are unnecessary for the `unzip` command, as Pi-Apps already has a function in the `api` script called `unzip`. This `unzip` function makes a line of output to say what is being extracted.

See the api line 2353.

Reply:
Resolved: [e31ca13](https://github.com/Botspot/pi-apps/pull/2052/commits/e31ca13b030484b318ec881494da2eb9d2a52ae9)